index.js

```diff
@@ -0,0 +1,132 @@
+import { Sequelize } from "sequelize";
+import AWS from "aws-sdk";
```

Comment:
can we update the import here to
import { SecretsManager } from 'aws-sdk', 

will this affect aws sdk initialisation

Reply:
I don't think changing that import will affect anything. will update it

Reply:
@muhammadtsneo  We cannot import secret manager like this!!